lib/ansible/modules/cloud/linode/linode.py

```diff
@@ -385,13 +384,8 @@ def linodeServers(module, api, state, name, alert_bwin_enabled, alert_bwin_thres
                 # Update linode Label to match name
                 api.linode_update(LinodeId=linode_id, Label='%s_%s' % (linode_id, name))
                 # Update Linode with Ansible configuration options
-                api.linode_update(LinodeId=linode_id, ALERT_BWIN_ENABLED=alert_bwin_enabled,
-                        ALERT_BWIN_THRESHOLD=alert_bwin_threshold, ALERT_BWOUT_ENABLED=alert_bwout_enabled,
-                        ALERT_BWOUT_THRESHOLD=alert_bwout_threshold, ALERT_BWQUOTA_ENABLED=alert_bwquota_enabled,
-                        ALERT_BWQUOTA_THRESHOLD=alert_bwquota_threshold, ALERT_CPU_ENABLED=alert_cpu_enabled,
-                        ALERT_CPU_THRESHOLD=alert_cpu_threshold, ALERT_DISKIO_ENABLED=alert_diskio_enabled,
-                        ALERT_DISKIO_THRESHOLD=alert_diskio_threshold, BACKUPWEEKLYDAY=backupweeklyday,
-                        BACKUPWINDOW=backupwindow, LPM_DISPLAYGROUP=displaygroup, WATCHDOG=watchdog)
+                api.linode_update(LinodeId=linode_id,
+                        LPM_DISPLAYGROUP=displaygroup, WATCHDOG=watchdog, **kwargs)
```

Comment:
I was able to fix this by
```
                api.linode_update(LinodeID=linode_id, Alert_bwin_enabled=alert_bwin_enabled,
                        Alert_bwin_threshold=alert_bwin_threshold, Alert_bwout_enabled=alert_bwout_enabled,
                        Alert_bwout_threshold=alert_bwout_threshold, Alert_bwquota_enabled=alert_bwquota_enabled,
                        Alert_bwquota_threshold=alert_bwquota_threshold, Alert_cpu_enabled=alert_cpu_enabled,
                        Alert_cpu_threshold=alert_cpu_threshold, Alert_diskio_enabled=alert_diskio_enabled,
                        Alert_diskio_threshold=alert_diskio_threshold, backupWeeklyDay=backupweeklyday,
                        backupWindow=backupwindow, lpm_displayGroup=displaygroup, watchdog=watchdog)
```
I'm confused why you kept LPM_DISPLAYGROUP and WATCHDOG as-is while you fixed the others. Can you add those to the kwargs like the rest? The [documentation](https://github.com/tjfontaine/linode-python/blob/master/linode/api.py#L413-L419) suggests those are expected to be lowercase too.